base/unicode/utf32.jl

```diff
@@ -11,7 +11,6 @@ sizeof(s::UTF32String) = sizeof(s.data) - sizeof(Char)
 
 const empty_utf32 = UTF32String(UInt32[0])
 
-utf32(x) = convert(UTF32String, x)
```

Comment:
In case someone else is reviewing this and gets confused like I did, this is a duplicate definition